template.rb

```diff
@@ -239,7 +225,7 @@ def index
 run 'rails g api_me:install'
 
 # Token Authentication Installation and Setup (token_authenticate_me and ember-authenticate-me)
-gem 'token_authenticate_me'
+gem 'token_authenticate_me', '>=0.4.2'
```

Comment:
As far as I know, it should try to pull the most recent version that matches. This was the one I tested with so I assumed it or higher would work.